sample-script/watchResource.js

```diff
@@ -0,0 +1,87 @@
+const { BotkitConversation } = require("botkit");
+
```

Comment:
Use the help plugin to add the watch commands to the help menu

Reply:
Done 👍